libcpuid/exports.def

```diff
@@ -30,3 +30,5 @@ cpu_clock_by_ic @26
 cpuid_get_total_cpus @27
 cpu_msr_driver_open_core @28
 cpuid_get_vendor @29
+cpu_rdmsr_range @30
+get_bits_value @31

```

Comment:
Do you really need to export this function? It's a small utility func, and having it globally in one place doesn't get any significant benefits. Especially if libcpuid is compiled as a shared library, it's actually better to have it locally in your executable, as then it can be inlined.
Also, naming. All libcpuid api starts with either 'cpu_' or 'cpuid_' (it's not very consistent, but basically 'cpu_*' functions are commands, "do something", while 'cpuid_*' are identification functions, "gimme that info").